src/components/ReplyCard/InReplyTo/InReplyToBase.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="in-reply-to-message-container">
+    <span class="in-reply-to-message-bar" :class="{ 'own-message': isOwnMessage }"></span>
+    <div class="in-reply-to-message">
+      <in-reply-to-text 
```

Comment:
Soon we will be able to reply all types with any other type, but they will be rendered differently. If we use only one component it will be too big with lots of condition cases.